packages/identity-manager/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@amplitude/identity-manager",
+  "version": "1.0.3",
+  "description": "Utility functions for the Amplitude JavaScript SDK",
```

Comment:
I think this was accidentally copied over from `utils`

Reply:
🤦 I thought I caught em all. Thanks!!